tools/importer-rest-api-specs/components/dataapigeneratorjson/templater_terraform_resource_definition.go

```diff
@@ -20,21 +20,23 @@ func buildTerraformResourceDefinition(resourceLabel string, input resourcemanage
 			Name:             input.DeleteMethod.MethodName,
 			TimeoutInMinutes: input.DeleteMethod.TimeoutInMinutes,
 		},
-		Description:                  input.Documentation.Description,
-		DisplayName:                  input.DisplayName,
-		ExampleUsage:                 input.Documentation.ExampleUsageHcl,
-		Generate:                     input.Generate,
-		GenerateIdValidationFunction: input.GenerateIdValidation,
-		GenerateModel:                input.GenerateModel,
-		GenerateSchema:               input.GenerateSchema,
-		Label:                        resourceLabel,
+		Description:  input.Documentation.Description,
+		DisplayName:  input.DisplayName,
+		ExampleUsage: input.Documentation.ExampleUsageHcl,
+		Generate:     input.Generate,
+		Label:        resourceLabel,
 		ReadMethod: dataApiModels.TerraformMethodDefinition{
 			Generate:         input.ReadMethod.Generate,
 			Name:             input.ReadMethod.MethodName,
 			TimeoutInMinutes: input.ReadMethod.TimeoutInMinutes,
 		},
 		ResourceIdName: input.ResourceIdName,
 		UpdateMethod:   nil,
+
+		// todo these are hardcoded to true in the C# generation. Does that hold true here?
```

Comment:
Yep, these aren't exposed right now but are cheap placeholders for when/if we need those in the future

However these should be hard-coded up one level, where the source data is passed in, so we can update these to use the existing `input.GenerateIdValidation` (etc) values and set this up one level if necessary?

Reply:
Done!